src/sbt-test/sbt-scalafix/relax-scalacOptions/test

```diff
@@ -1,22 +1,45 @@
-# we have 2 fatal warnings preventing compilation
+# when the build does not contain any flag relaxed on scalafix invocation
+# check that regular compilation benefits froma prior direct scalafix invocation ...
+-> scalafix --check RemoveUnused
+-> checkLastCompilationCached
+> compile
+> checkLastCompilationCached
```

Comment:
users with no fatal warnings calling scalafix before compile no longer have to compile everything twice